start-site/src/main/resources/application.yml

```diff
@@ -199,12 +207,26 @@ initializr:
       spring-releases:
         name: Spring Releases
         url: https://repo.spring.io/release
+      spring-milestones:
```

Comment:
You don't need this. `spring-milestones` and `spring-snapshots` are automatically available.